ptsprojects/zephyr/gatt.py

```diff
@@ -502,6 +502,10 @@ def test_cases_server(ptses):
                   cmds=pre_conditions_1 +
                        [TestFunc(btp.gap_set_io_cap, IOCap.display_only)],
                   generic_wid_hdl=gatt_wid_hdl),
+        ZTestCase("GATT", "GATT/SR/GAS/BV-03-C",
```

Comment:
@ryanjh as we discussed there is a PTS defect:
The lower should be writing 0b11111001 to the Client Supported Features characteristic,
but instead it's writing 0xFFFF.

I will approve the PR once you open an issue with the SIG

Reply:
Case created as the link.

https://bluetooth.service-now.com/ess/case_detail.do?sysparm_document_key=x_bsig_case_manage_case,d8ff61591b80a410186a4000cd4bcb4a